packages/scandipwa/src/component/AddToCart/AddToCart.container.js

```diff
@@ -169,6 +169,13 @@ export class AddToCartContainer extends PureComponent {
         const isValid = typeId === PRODUCT_TYPE.grouped || inRange;
 
         if (!isValid) {
+            if (minQty > maxQty) {
+                showNotification('info', __('The requested qty is not available!', minQty));
```

Comment:
You have useless second attribute of the `__()` function here.
If you want to render it, put a `%s` placeholder to the string:
```javascript
__('Value: %s', value);
```
Or remove the second argument, if it's not needed.

Reply:
done